src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -206,6 +207,54 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     fromAddress: fromAddress ?? null,
   })
 
+  const { amountCryptoBaseUnitMinusFees, amountCryptoHumanMinusFees } = useMemo(() => {
```

Comment:
What will this show for a value between 0 and estimated fees above? Will this yield a negative number? Probably edge-case since the outbound fees guard *should* catch it at deposit time, feel free to close this comment if so

Reply:
As you said, should be caught at deposit time, but we would effectively show a negative number, very edgy as we already have a guard before